utility/doc/CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.0.0.7] - 2022-10-12
+## [0.0.0.7] - 2022-10-14
+
+- Added session interfaces and diagrams
```

Comment:
Should this be a separate release?

Reply:
I've been doing this with 'releases' that are minor and within a few days. Considering we don't have a process for this - I think this is fine as is